internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -1147,10 +1147,15 @@ func resourceCosmosDbAccountDelete(d *pluginsdk.ResourceData, meta interface{})
 		return err
 	}
 
-	if _, err := client.Delete(ctx, id.ResourceGroup, id.Name); err != nil {
+	future, err := client.Delete(ctx, id.ResourceGroup, id.Name)
+	if err != nil {
 		return fmt.Errorf("deleting CosmosDB Account %q (Resource Group %q): %+v", id.Name, id.ResourceGroup, err)
 	}
 
+	if err := future.WaitForCompletionRef(ctx, client.Client); err != nil {
+		return fmt.Errorf("waiting for creation/update of %q: %+v", id, err)
+	}
```

Comment:
@heiazuo this LRO isn't used per below since the API is broken, what happens if the manual poller below is removed?

Reply:
> @heiazuo this LRO isn't used per below since the API is broken, what happens if the manual poller below is removed?

Thanks @tombuildsstuff , Try to remove this module, and then test, it will not affect